src/styles/card.scss

```diff
@@ -0,0 +1,86 @@
+.card {
+  width: 200px;
+  height: 406px;
+  box-sizing: border-box;
+
+  // border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+
+  // background-color: yellow;
+  // display: flex;
+  // justify-content: space-between;
+  // flex-direction: column;
+  // align-items: center;
+
+  &__logo {
+    width: 160px;
+    height: 134px;
+    background-image: url(../images/imac.jpeg);
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: contain;
+    margin-bottom: 40px;
+  }
+
+  &__description {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $text-color;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    width: 100%;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: left;
+    color: $gray-color;
+    margin-bottom: 16px;
+  }
+
+  &__price {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+
+    &-name {
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+      text-align: left;
+      color: $gray-color;
+    }
+
+    &-amount {
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      text-align: right;
+      color: $text-color;
+    }
+  }
+
+  &__button {
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 40px;
+    text-align: center;
+    color: #fff;
```

Comment:
The `color` property should use a variable for consistency and maintainability, especially since you've used variables for other colors in the file. If white is a commonly used color in the design, consider defining it as a variable.